ts/python/mod.ts

```diff
@@ -0,0 +1,114 @@
+import { Environment, mkEnvironment } from "../environment.ts";
+import { mkPackage, Package } from "../package.ts";
+import { mkProject, Project } from "../project.ts";
+import { nixSource } from "../internal/utils.ts";
+import { nixRaw } from "../nix.ts";
+import * as pythonInterpreters from "../internal/nixpkgs/pythonInterpreters/mod.ts";
+import { parsePyprojectToml } from "./utils.ts";
+export { plugin as vite } from "../javascript/vite.ts";
+
+export const interpreters = pythonInterpreters;
+
+/**
+ * Creates a pyproject based python garn Project.
+ */
+export function mkPythonProject(args: {
+  // fields like name, version, description are read from pyproject.toml
```

Comment:
Maybe this should go in the user-facing documentation two lines above rather than here? (And the lines below I think should be removed - I think what should come from the language-specific config file and what should be specified manually is an overarching policy that we still have to settle on more deliberately?)

Reply:
fixed the documentation placement via 56c2e3245609dc7cdd46ded683de46e33c3568fd